src/GraphQL/Utilities/AstPrinter.cs

```diff
@@ -368,6 +405,8 @@ public AstPrintVisitor()
                 c.Print(p => $"{p.Arg(x => x.Name)}: {p.Arg(x => x.Value)}");
             });
 
+            Config<AnyValue>(c => c.Print(x => throw new InvalidOperationException("Cannot print AnyValue representations.")));
```

Comment:
Somewhere there is such a requirement?

Reply:
I don't know how `AnyValue` is used for federation, but I know it's needed.  My best guess is that it's a placeholder for a part of the AST that's not part of the local instance, or something.  In any case, we certainly can't print it, without knowing what type it's supposed to be.

I figured throwing an error would be better than before, where it silently skipped printing the value (probably resulting in an invalid syntax).

Reply:
Ok